### PR TITLE
fix default value for *_credentials_secure_logging in README

### LINKED ISSUE #990: No consistency between `README.md` and `defaults/main.yml` in `controller_credentials` role
## Summary

The default value of `controller_configuration_credentials_secure_logging` in [`controller_credentials/defaults/main.yml`](https://github.com/redhat-cop/infra.aap_configuration/blob/3.1.0/roles/controller_credentials/defaults/main.yml#L4) is `"{{ aap_configuration_secure_logging | default(true) }}"`.

But [`README.md`](https://github.com/redhat-cop/infra.aap_configuration/tree/3.1.0/roles/controller_credentials#secure-logging-variables) says default is `false`.

Which is correct?

## Issue Type

- Bug Report

## Ansible, Collection, Controller details

N/A (Document only)

## OS / ENVIRONMENT

N/A (Document only)

## Desired Behavior

N/A (Document only)

## Actual Behavior

N/A (Document only)

## STEPS TO REPRODUCE
N/A (Document only)

### PR DESCRIPTION
**Pull request by akira6592**
# What does this PR do?

fix default value for `*_credentials_secure_logging` in README.

# How should this be tested?

N/A (Document only)

# Is there a relevant Issue open for this?

resolves #990 

# Other Relevant info, PRs, etc

N/A (Document only)

### CODE REVIEW

File: roles/controller_credentials/README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Enabling this will enforce configurtion without specifying every option in the c
 
 |Variable Name|Default Value|Required|Description|
 |:---:|:---:|:---:|:---:|
-|`controller_configuration_credentials_enforce_defaults`|`false`|no|Whether or not to enforce default option values on only the applications role|
+|`controller_configuration_credentials_enforce_defaults`|`true`|no|Whether or not to enforce default option values on only the applications role|
```
Comment (djdanielsson) [inline roles/controller_credentials/README.md]: I believe you meant, controller_configuration_credentials_secure_logging not controller_configuration_credentials_enforce_defaults

**Review by djdanielsson — CHANGES_REQUESTED** [review]

File: roles/controller_credentials/README.md (hunk 1 above)
Comment (akira6592) [reply roles/controller_credentials/README.md]: @djdanielsson Thank you for review. I fixed it.

**Review by djdanielsson — APPROVED** [review]
LGTM

**Comment by akira6592** [comment]
Thank you for merging!

**Pull request merged**